src/TwilioVideo.android.js

```diff
@@ -292,7 +302,8 @@ class CustomTwilioVideoView extends Component {
       'onParticipantDisabledAudioTrack',
       'onStatsReceived',
       'onNetworkQualityLevelsChanged',
-      'onDominantSpeakerDidChange'
+      'onDominantSpeakerDidChange',
+      'onLocalParticipantH246Supported'
     ].reduce((wrappedEvents, eventName) => {
```

Comment:
What cause the even to be triggered at the correct time with the correct arguments?

Reply:
https://github.com/brycnguyen/react-native-twilio-video-webrtc-1/pull/1/files#diff-aad5b5dfca4228dee4cff124b7f644acd8b8dfa6c1fcf9d2bcfb44b8052c4675R530